ironfish-cli/src/commands/accounts/pay.ts

```diff
@@ -95,7 +95,7 @@ export class Pay extends IronfishCommand {
       }
     }
 
-    if (!fee || Number.isNaN(Number(fee))) {
+    if (!fee || Number.isNaN(fee)) {
```

Comment:
`Number.isNaN` just checks the type of the input. In this case it will always return false because fee is type string.

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN